### PR TITLE
[appaloosa] add description support

### PR DESCRIPTION
**Pull request by sylvek**
allow to setup the description of this app'.
it may be helpful to comunicate the release note for example.

### CODE REVIEW

**Comment by ohwutup** [comment]
Hey @sylvek! Would you mind rebasing on master and pushing up again to see if that will fix the CI failure?

**Comment by sylvek** [comment]
ok :)

File: fastlane/lib/fastlane/actions/appaloosa.rb [hunk 1]
```diff
@@ -104,6 +104,7 @@ def self.upload_on_appaloosa(api_key, store_id, binary_path, screenshots, group_
         req.body = { store_id: store_id,
                      api_key: api_key,
                      mobile_application_update: {
+                       description: description,
```
Comment (ohwutup) [inline fastlane/lib/fastlane/actions/appaloosa.rb]: If `description` is nil here, will this request body cause an issue on Appaloosa's end?

**Comment by ohwutup** [comment]
Just added one more little question for you @sylvek !

File: fastlane/lib/fastlane/actions/appaloosa.rb (hunk 1 above)
Comment (sylvek) [inline fastlane/lib/fastlane/actions/appaloosa.rb]: hey @ohwutup thank you for your time.
i read the current api v2 https://appaloosa-store.com/api and the description field is optional.

Comment (ohwutup) [inline fastlane/lib/fastlane/actions/appaloosa.rb]: So if it is optional, maybe we should only insert the `description` key into the dictionary if there is a description present to fill the value. I would be a little worried to pass `nil` and if that would wipe out some data that may already be setup in Appaloosa, or just to not add unnecessary parameters.

Comment (sylvek) [inline fastlane/lib/fastlane/actions/appaloosa.rb]: you are right i guess, but all other field are optionals too, so it is why i do that.

Comment (ohwutup) [inline fastlane/lib/fastlane/actions/appaloosa.rb]: Oops! Good point! This is great then! Sorry for the back and forth.

**Comment by chrsvl** [comment]
Many thanks from the Appaloosa dev team ;)

**Pull request merged**